feat: ROOT-179: Prompts-related endpoints in SDK

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/17985822767)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/585
<!-- Follow Merge -->

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17654081251

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17683805463

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-179'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17683805463

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17737642200

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-179'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17737642200

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17737946848

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17744600516

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-179'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17744600516

matt-bernstein commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17801372761)
> Successfully merged:  32 files changed, 906 insertions(+), 405 deletions(-)

matt-bernstein committed:
Merge branch 'develop' into 'fb-ROOT-179'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17801372761

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17859703510

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-179'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17859703510

matt-bernstein committed:
lockfile

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17861829897

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17865497452

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17867973319

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-179'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17867973319

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17868129603

matt-bernstein commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17951394192)
> Successfully merged:  54 files changed, 936 insertions(+), 553 deletions(-)

matt-bernstein committed:
Merge branch 'develop' into 'fb-ROOT-179'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17951394192

matt-bernstein commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17954543531)
> Successfully merged: Already up to date.

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17954599001

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17954727730

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-179'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17954727730

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17985727875

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-179'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17985727875

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17985822767

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio-enterprise/actions/runs/17986546152
